DSharpPlus.SlashCommands/Contexts/AutocompleteContext.cs

```diff
@@ -36,6 +38,44 @@ public class AutocompleteContext
         /// </summary>
         public DiscordInteraction Interaction { get; internal set; }
 
+        /// <summary>
+        /// Gets the client for this interaction.
+        /// </summary>
+        public DiscordClient Client { get; internal set; }
+
+        /// <summary>
+        /// Gets the guild this interaction was executed in.
+        /// </summary>
+        public DiscordGuild Guild { get; internal set; }
+
+        /// <summary>
+        /// Gets the channel this interaction was executed in.
+        /// </summary>
+        public DiscordChannel Channel { get; internal set; }
+
+        /// <summary>
+        /// Gets the user which executed this interaction.
+        /// </summary>
+        public DiscordUser User { get; internal set; }
+
+        /// <summary>
+        /// Gets the member which executed this interaction, or null if the command is in a DM.
+        /// </summary>
+        public DiscordMember Member
+            => this.User is DiscordMember member ? member : null;
+
+        /// <summary>
+        /// Gets the slash command module this interaction was created in.
+        /// </summary>
+        public SlashCommandsExtension SlashCommandsExtension { get; internal set; }
+
+        /// <summary>
+        /// <para>Gets the service provider.</para>
+        /// <para>This allows passing data around without resorting to static members.</para>
+        /// <para>Defaults to null.</para>
+        /// </summary>
+        public IServiceProvider Services { get; internal set; } = new ServiceCollection().BuildServiceProvider(true);
+
```

Comment:
Not really familiar with docs style for this repo overall (in fact I copied docs stuff from other source file in repo since I have no idea what to put for docs). I'll leave this open for docs suggestion especially by maintainers